settings.py

```diff
@@ -82,3 +82,7 @@
 PR_STALE_HOURS = 36
 
 API_COOLDOWN_RESET_PADDING = 30
+
+# list of autocracy members
+with open("data/autocracy.txt", "r") as f:
```

Comment:
I kind of like the file for these long lists. We do the same for lists of emojis for example...